Description: Temporarily disable B-D on log to unblock build
 This crate and its B-D on log is in a chain that bidirectionally blocks regex
 and regex-automata. No-op +logging to try to unblock it.
Last-Update: 2023-10-02
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -69,19 +69,19 @@
 optional = true
 package = "rustc-std-workspace-core"
 
-[dependencies.log]
-version = "0.4.20"
-optional = true
+#[dependencies.log]
+#version = "0.4.20"
+#optional = true
 
-[dev-dependencies.quickcheck]
-version = "1.0.3"
-default-features = false
+#[dev-dependencies.quickcheck]
+#version = "1.0.3"
+#default-features = false
 
 [features]
 alloc = []
 default = ["std"]
 libc = []
-logging = ["dep:log"]
+#logging = ["dep:log"]
 rustc-dep-of-std = [
     "core",
     "compiler_builtins",
--- a/src/macros.rs
+++ b/src/macros.rs
@@ -12,9 +12,9 @@
 }
 
 macro_rules! debug {
-    ($($tt:tt)*) => { log!(log::debug!($($tt)*)) }
+    ($($tt:tt)*) => { }// log!(log::debug!($($tt)*)) }
 }
 
 macro_rules! trace {
-    ($($tt:tt)*) => { log!(log::trace!($($tt)*)) }
+    ($($tt:tt)*) => { }// log!(log::trace!($($tt)*)) }
 }
